### PR TITLE
Add commands to list volumes and snapshots

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

File: commands/volumes/snapshots.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	for _, v := range resp.Data {
+		data = append(data, []string{
+			strconv.Itoa(v.Attributes.FromSnapshotNumber),
+			strconv.Itoa(v.Attributes.SequenceNumber),
+			v.Attributes.Status,
+			v.Type,
+		})
+	}
+
+	columns := []string{"From", "Sequence", "Type", "Status"}
```
Comment (elliott-king) [inline commands/volumes/snapshots.go:79]:
```suggestion
	columns := []string{"From", "Sequence", "Status", "Type"}
```

Here's what I see:
```
  FROM  SEQUENCE  TYPE   STATUS
---------------------------------------
  4     5         DONE   snapshot-set
  3     4         DONE   snapshot-set
  1374  3         DONE   snapshot-set
  1     2         ERROR  snapshot-set
  1370  1         DONE   snapshot-set

```

**Review by elliott-king — CHANGES_REQUESTED** [review]
When there are no snapshots/volumes, the UI could be clearer:
```
;./shipyard get snapshots --env 3879ecad-fe10-4452-8235-dd3b1ec1c256
  FROM  SEQUENCE  TYPE  STATUS
--------------------------------

```

File: commands/volumes/snapshots.go (hunk 1 above)
Comment (maxsokolovsky) [reply commands/volumes/snapshots.go]: Adjusted. Also, now displaying a "not found" message for when there are no environments and no snapshots associated with an environment.

**Review by elliott-king — APPROVED** [review]
Please don't revert & force-push. It's hard to see what the newest changes are.

**Pull request merged**